core/src/main/kotlin/plugins/HTTP.kt

```diff
@@ -19,12 +19,23 @@
 
 package org.eclipse.apoapsis.ortserver.core.plugins
 
+import io.ktor.http.HttpHeaders
+import io.ktor.http.HttpMethod
 import io.ktor.server.application.Application
 import io.ktor.server.application.install
+import io.ktor.server.plugins.cors.routing.CORS
 import io.ktor.server.plugins.defaultheaders.DefaultHeaders
 
 fun Application.configureHTTP() {
     install(DefaultHeaders) {
         header("X-Engine", "Ktor") // will send this header with each response
     }
+    install(CORS) {
```

Comment:
We usually prefer to have an empty line before new code blocks / between previous and next code blocks.

Reply:
... but actually, this does not necessarily seem to be the case for the ORT server code base, also see https://github.com/eclipse-apoapsis/ort-server/pull/81/files.